### ISSUE TITLE
 virtual box has a white background in dark colorscheme

### ISSUE THREAD

**Issue by BurningSmile** [actionless/oomox#53]
Screenshot of the problem
https://i.imgur.com/BVyljqq.png

Settings being used
https://i.imgur.com/sHSwQ36.png

Tweak tool settings
https://i.imgur.com/XpfPCOW.png

Any ideas what options in here are causing this?

Thanks

**Comment by actionless** [comment]
could you please try with the latest git version?

if it will be still bad you can try setting spotify theme variables manually (it's not implemented in GUI yet), see an example in this commit: https://github.com/actionless/oomox/commit/a0b564c416f5130bd5df66dacce34ff9b4dcfa63

**Comment by BurningSmile** [comment]
Just removed the normal version and installed the git version, then rebuilt the theme. The problem is still there. 

How do I use thous additions you implemented via the cli?

**Comment by actionless** [comment]
```diff
--- a/oomoxify.sh
+++ b/oomoxify.sh
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
 set -ueo pipefail
```

and paste first ~100 lines of the output

**Comment by BurningSmile** [comment]
`/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
IS DARK
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
/home/david/oomox/scripts/darker.sh: line 7: bc: command not found
about.spa
ad.spa
album.spa
artist-chart.spa
artist.spa
browse.spa
buddy-list.spa
chart.spa
collection-album.spa
collection-artist.spa
collection-songs.spa
collection.spa
concerts.spa
daily-mix-hub.spa
discover.spa
error.spa
findfriends.spa
full-screen-modal.spa
genre.spa
glue-resources.spa
hub.spa
licenses.spa
lyrics.spa
messages.spa
playlist-desktop.spa
playlist-folder.spa
profile.spa
radio-hub.spa
search.spa
settings.spa
share.spa
station.spa
stations.spa
suggest.spa
zlink-queue.spa
zlink.spa
zlogin.spa
`

**Comment by actionless** [comment]
updated `oomox-git` package in AUR

**Comment by BurningSmile** [comment]
Alright, it will probably take an hour or so to hit the mirrors as it has not showed up as an updated package in the AUR yet.

I will report back what happens after the package can be updated.

**Comment by actionless** [comment]
i've just checked with `yaourt` -- 
![2016-11-23--1479864162_716x119_scrot](https://cloud.githubusercontent.com/assets/1655669/20548427/b8b2399e-b123-11e6-8a6f-7a9f3f17d780.png)

**Comment by BurningSmile** [comment]
I ran yaourt oomox-git and reinstalled the package. (I could not get it to update, using the mirrors from the Arch Linux mirrorlist generator)

I then removed the old oomox directory and redid the spotify stall then ran the ./change_color.sh numix command. The output has changed the the following but the problem persists.

output

http://pastebin.com/raw/8VECaQkY

**Comment by actionless** [comment]
you need to run `oomoxify` instead of `change_color` and also specify full path to your theme (or otherwise default `numix `will be used)

**Comment by BurningSmile** [comment]
I ran oomoxify then selected the custom numix theme and ran the the apply numix theme.

It worked.

Thanks for your work on all this btw, it is much appreciated :) 

Also I noticed that 1 specific application won't match with the new numix theme I made through oomox. (essentially default just changed and ran to force applications to use a dark interface) 

The problem is virtual box which still has a white background. (such as Vlc and obs.) This problem was not present on my last install. I followed the steps listed in [this Arch wiki page.](https://wiki.archlinux.org/index.php/Uniform_look_for_Qt_and_GTK_applications)
